index.js

```diff
@@ -20,12 +26,13 @@ if (!parameters.options) {
   return;
 }
 
-const options = JSON.parse(fs.readFileSync(absolutePath(parameters.options)));
+const options = JSON.parse(fs.readFileSync(/* absolutePath( */parameters.options)/* ) */);
```

Comment:
Should avoid commented implementations (source control is a better to see changes)